tasks/Task.cpp

```diff
@@ -26,16 +26,21 @@ Task::~Task()
 
 bool Task::configureHook()
 {
+    if (!_io_port.get().empty())
+      lamps.openURI(_io_port.value());
+    else
+      throw std::runtime_error("io_port property not set");
```

Comment:
This is actually valid, one can be fed data through ports when using `iodrivers_base::Task`. Valid combinations of ports connected and call to openURI is validated in [`iodrivers_base::Task::configureHook`](https://github.com/rock-drivers/drivers-orogen-iodrivers_base/blob/master/tasks/Task.cpp#L48)